packages/next/server/next-server.ts

```diff
@@ -1079,13 +1075,6 @@ export default class NextNodeServer extends BaseServer {
     }))
   }
 
-  protected getEdgeRoutes(): RoutingItem[] {
-    const edgeFunctions = this.getEdgeFunctions()
-    const middleware = this.getMiddleware()
-
-    return edgeFunctions.concat(middleware ? [middleware] : [])
-  }
-
```

Comment:
Not used anywhere since https://github.com/vercel/next.js/commit/9a1b756defbdc735a136504aae7fa1a6212eefb4#diff-ea09101c272788873ce720075e4839f0254179af0a07259a107752cafb193949L1213